enterprise-suite/install-es.sh

```diff
@@ -25,7 +25,8 @@ function usage() {
     docvar ES_CHART "Chart name to install from the repository"
```

Comment:
Is there a reason for controlling things with env vars as opposed to using command line flags?  Just curious.

Reply:
1. Parsing flags in bash is horrible and will add a lot of complexity to the script
2. I want to just pass through the arguments to helm so we don't need to duplicate helm - we can't do that easily if we use command line flags.